canvas_api/src/types/responses/conversations.rs

```diff
@@ -0,0 +1,33 @@
+use chrono::{DateTime, Utc};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Conversation {
+    pub id: u64,
+    pub subject: String,
+    pub last_message: String,
+    pub subscribed: bool,
+    pub private: bool,
+    pub starred: bool,
+    pub tags: Option<Vec<String>>,
+    pub message: Vec<ConversationMessage>
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ConversationMessage {
+    pub id: u64,
+    pub body: String,
+    pub author_id: String,
+    pub generated: bool,
+    pub media_comment: String,
```

Comment:
Seperate type + Nullable

> Audio/video comment data for this message (if applicable). Fields include: display_name, content-type, media_id, media_type, url

```suggestion
    // pub media_comment: Option<()>,
```